docs/pipeline-components-and-applications/enrichment-components/enrich-kafka/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Enrich Kafka (cloud agnostic)"
+date: "2022-10-03"
+sidebar_position: 10
```

Comment:
Could you move this to be just above RabbitMQ? I think having the cloud agnostic options next to each other would look nicer

Reply:
sure

Reply:
just added a new commit, they were mostly same value, please let me know if it is okay @stanch